Pod/Classes/PagingMenuController.swift

```diff
@@ -321,8 +323,14 @@ extension PagingMenuController: UIScrollViewDelegate {
         
         move(toPage: nextPage)
     }
-    
+
+    public func scrollViewWillBeginDragging(_ scrollView: UIScrollView) {
```

Comment:
I've just tried, and it seems this is not triggered when scrolling *starts*.

https://developer.apple.com/reference/uikit/uiscrollviewdelegate indicates that this should be used:

func scrollViewWillBeginDragging(UIScrollView)